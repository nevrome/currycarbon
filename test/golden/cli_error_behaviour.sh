# expression that yields only zero
## no files requested 
currycarbon "uncalC14(3200,30) * rangeBP(3000,2800)"
## hdr file requested
currycarbon "uncalC14(3200,30) * rangeBP(3000,2800)" --hdrFile /dev/null
## samples file requested
currycarbon "uncalC14(3200,30) * rangeBP(3000,2800)" --samplesFile /dev/null -n 3
## hdr & samples file requested
currycarbon "uncalC14(3200,30) * rangeBP(3000,2800)" --hdrFile /dev/null --samplesFile /dev/null -n 3