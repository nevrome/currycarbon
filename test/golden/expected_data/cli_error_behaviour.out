currycarbon v0.4.0.0 (UTF-8)
Method: Bchron {distribution = StudentTDist {ndf = 100.0}}
Curve: IntCal20
Calibrating...
<!> Error: Invalid CalPDF for refinement, either because all densities are 0 or one density is > 1
Done.
currycarbon v0.4.0.0 (UTF-8)
Method: Bchron {distribution = StudentTDist {ndf = 100.0}}
Curve: IntCal20
Calibrating...
<!> Error: Invalid CalPDF for refinement, either because all densities are 0 or one density is > 1
<!> Error: Can not create --hdrFile
Done.
currycarbon v0.4.0.0 (UTF-8)
Method: Bchron {distribution = StudentTDist {ndf = 100.0}}
Curve: IntCal20
Calibrating...
<!> Error: Invalid CalPDF for refinement, either because all densities are 0 or one density is > 1
<!> Error: Invalid CalPDF for random age sampling, either because all densities are 0 or one density is > 1
<!> Error: Can not create --samplesFile
Done.
currycarbon v0.4.0.0 (UTF-8)
Method: Bchron {distribution = StudentTDist {ndf = 100.0}}
Curve: IntCal20
Calibrating...
<!> Error: Invalid CalPDF for refinement, either because all densities are 0 or one density is > 1
<!> Error: Can not create --hdrFile
<!> Error: Invalid CalPDF for random age sampling, either because all densities are 0 or one density is > 1
<!> Error: Can not create --samplesFile
Done.
currycarbon v0.4.0.0 (UTF-8)
Method: Bchron {distribution = StudentTDist {ndf = 100.0}}
Curve: IntCal20
Calibrating...
<!> Error: Invalid CalPDF for refinement, either because all densities are 0 or one density is > 1
Done.
CalEXPR: [1] (1p:3200±30BP * 1P:3000BP-2800BP)
CalEXPR: [1] (1p:3200±30BP * 1P:3000BP-2800BP)
CalEXPR: [1] (1p:3200±30BP * 1P:3000BP-2800BP)
CalEXPR: [1] (1p:3200±30BP * 1P:3000BP-2800BP)
CalEXPR: [1] (1p:3200±30BP * 1P:3000BP-2800BP)
